main/src/library/Fixpoint/Ast/BodyPredicate.flix

```diff
@@ -58,19 +59,20 @@ namespace Fixpoint/Ast {
             };
             match body {
                 case BodyAtom(predSym, Relational, p, f, terms) =>
-                    "${polarityPrefix(p)}${fixityPrefix(f)}${predSym}(${commaSeparate(terms)})"
+                    "${polarityPrefix(p)}${fixityPrefix(f)}${predSym}(${terms |> List.map(toString) |> String.intercalate(", ")})"
                 case BodyAtom(predSym, Latticenal(_), p, f, terms) =>
-                    let len = Array.length(terms);
-                    let k = terms[0..len-1];
-                    let l = terms[len-1];
-                    "${polarityPrefix(p)}${fixityPrefix(f)}${predSym}(${commaSeparate(k)}; ${l})"
+                    let (k, lList) = List.splitAt(List.length(terms)-1, terms);
```

Comment:
I propose something like `keys` and `lattice` or `ks` and `l` or similar or `keyTerms` and `latticeTerm`?